zebra-chain/src/transparent/utxo.rs

```diff
@@ -54,6 +54,24 @@ pub struct OrderedUtxo {
     pub tx_index_in_block: usize,
 }
 
+impl<'a> From<&'a OrderedUtxo> for &'a Utxo {
+    fn from(ordered_utxo: &'a OrderedUtxo) -> Self {
+        &ordered_utxo.utxo
+    }
+}
+
+impl From<OrderedUtxo> for Utxo {
+    fn from(ordered_utxo: OrderedUtxo) -> Self {
+        ordered_utxo.utxo
+    }
+}
+
```

Comment:
These aren't currently used anywhere.

```suggestion
```

Reply:
I think it's ok to delete them, `ordered_utxo.utxo` is clearer and shorter.